.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,10 +7,9 @@ linked together.
 
 ### Checklist
 
-- [ ] Did you run `npm test`?
+- [ ] Have you added unit and functional tests as appropriate?
```

Comment:
How can we add functional tests when we don't have the functional tests set up yet?